### PR TITLE
@/engineeringblog GA analytics

### PR DESCRIPTION
**Pull request by kilemensi**
## Description

This PR adds GA to `engineeringblog` app.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots

Screenshot 2024-09-18 at 12 35 37

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: apps/engineeringblog/content/site/settings.mdx [hunk 1]
```diff
@@ -35,4 +35,6 @@ connect:
       url: https://www.instagram.com/code4africa__/
     - platform: Github
       url: https://github.com/CodeForAfrica
+analytics:
+  analyticsId: "G-TN1WGZVHLE"
```
Comment (DavidTheProgrammer) [inline apps/engineeringblog/content/site/settings.mdx:39]: Where can I get more information about this. Does the tsx layout have access to this post compilation as it's an mdx file? A little lost on how the pieces fit together

Comment (kilemensi) [reply apps/engineeringblog/content/site/settings.mdx:39]: Where is the real `MumaTheProgrammer` and what have you done to him @DavidTheProgrammer ?

  1. The mdx file is loaded [here](https://github.com/CodeForAfrica/ui/blob/442957676567702cd4da7dcba78e00fe809278fd/apps/engineeringblog/lib/data/index.ts#L123-L126), and
  2. It's added to the layout [here](https://github.com/CodeForAfrica/ui/blob/442957676567702cd4da7dcba78e00fe809278fd/apps/engineeringblog/app/layout.tsx#L23-L24).

Comment (DavidTheProgrammer) [reply apps/engineeringblog/content/site/settings.mdx:39]: Lol i'll blame it on the flu. But looks good 🚀

**Review by DavidTheProgrammer — APPROVED** [review]
Looks good 🚀

**Pull request merged**